codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -71,7 +80,22 @@
      *        will be called.
      */
     public HttpObjectAggregator(int maxContentLength) {
+        this(maxContentLength, false);
+    }
+
+    /**
+     * Creates a new instance.
+     * @param maxContentLength
+     *        the maximum length of the aggregated content in bytes.
+     *        If the length of the aggregated content exceeds this value,
+     *        a {@link TooLongFrameException} will be raised.
+     * @param closeOnExpectationFailed If a 100-continue response is detected but the content length is too large
```

Comment:
can you use the same spacing for maxContentLength and closeOnExceptionFailed description ?